### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by VR56**

### CODE REVIEW

File: Assignment_1_Problem_1.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Problem 1
+// Prolem statement 
+// http://www.codewars.com/kata/printing-array-elements-with-comma-delimiters
+
+// Solution
+
+function printArray(array) {
```
Comment (SaiTeja2904) [inline Assignment_1_Problem_1.js]: Explore Array inbuilt methods. This can be solved using join().

File: Assignment_1_Problem_4.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Problem 4
+// Prolem statement 
+// http://www.codewars.com/kata/transportation-on-vacation
+
+// Solution
+
+function rentalCarCost(days) {
+    var TotalPrice = days*40;
```
Comment (SaiTeja2904) [inline Assignment_1_Problem_4.js]: Use Camel Case notation for using variables.

File: Assignment_1_Problem_4.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+function rentalCarCost(days) {
+    var TotalPrice = days*40;
+    if (days >= 3 && days < 7){  //If days are more than or equal to 3 and less than 7
+        return TotalPrice - 20;  //Discounted price
+    }
+    else if (days >= 7){         //If days are greate than equal to 7
+        return TotalPrice - 50;  //Discounted price
+    }
+    else if (days < 3){          //If days are less than 3
+        return days * 40;        //Price
```
Comment (SaiTeja2904) [inline Assignment_1_Problem_4.js]: Try avoiding multiple return statements.

File: Assignment_2_Problem_2.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+//Assignment 2
+// Problem 2
+// Prolem statement 
+// http://www.codewars.com/kata/get-the-middle-character
+
+// Solution
+
+function getMiddle(string) {
+
+    if (string.length % 2 === 0) { //if even length string is encountered
```
Comment (SaiTeja2904) [inline Assignment_2_Problem_2.js]: Don't compute str.length multiple times instead assign it to variable and use.

File: Assignment_2_Problem_3.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+//Assignment 2
+// Problem 3
+// Prolem statement 
+// http://www.codewars.com/kata/partition-on
+
+// Solution
+
+function partitionOn(pred, items) {
```
Comment (SaiTeja2904) [inline Assignment_2_Problem_3.js]: Explore inbuilt array methods. Try solving with them.

**Comment by VR56** [comment]
I have implemented the required changes. Please review the code

**Pull request merged**